src/lib.rs

```diff
@@ -426,6 +426,7 @@ pub mod marshal;
 pub mod sync;
 pub mod panic;
 pub mod prelude;
+pub mod py_backed;
```

Comment:
I understand why `py_backed` is in some sense more correct than `pybacked`, but since we went for `pyclass`, I think I would value consistency higher here.